freeipa.spec.in

```diff
@@ -625,7 +632,7 @@ If you are installing an IPA server, you need to install this package.
 Summary: IPA integrated DNS server with support for automatic DNSSEC signing
 BuildArch: noarch
 Requires: %{name}-server = %{version}-%{release}
-Requires: bind-dyndb-ldap >= 11.2-2
+Requires: bind-dyndb-ldap >= 11.11
```

Comment:
On fedora 41 we only have [bind-dyndb-ldap-11.10-33.fc41](https://koji.fedoraproject.org/koji/buildinfo?buildID=2655860)

Reply:
Same here.